### ISSUE TITLE
Loss, Val Acc history for training

### ISSUE THREAD

**Issue by bmmidei** [ac4391/EssayGrader#7]
Include loss and validation history during training. This can be used to plot training loss and val acc as a function of the number of epochs.
